requirements.txt

```diff
@@ -21,6 +21,7 @@ papermill
 # Pin pytest due to VS Code issue
 pytest==5.0.1
 pytest-cov
+pytest-mpl>=0.11
```

Comment:
We should devise a way of testing our packages with-and-without the plotting extras

Reply:
in the test configs, you can `SkipTest` a test or a file if a certain criteria is not satisfied. As an example you can see: https://github.com/scikit-learn/scikit-learn/pull/14675

Reply:
I've added a pytest.skipif, but are we okay with leaving it in requirements.txt?

Reply:
it's not the heaviest and the worst item in the requirement.txt :P I'd leave it there :)

But you may want to add a CI to run the tests w/o the plotting deps installed, if it's not already there.

Reply:
We have a pipeline running without matplotlib, but not without pytest-mpl. I'll look into it

Reply:
ok then, I don't think one w/o `pytest-mpl` is needed then.